### DISCUSSION TITLE
Can we get the true Day andtimeStamp info from the test_x dataset

### DISCUSSION THREAD

**Discussion by oneday88** [PaddlePaddle/PaddleSpatial#123] [BaiduKDDCup2022] [answered]
In my understanding, the stride time step is randomly sampled, hence the starting time of the testing data will be any time in the data.  for the toy data "test_x", it seems  the  starting timestamp of the testing data are aligned to Day 1 and timestamp 00:00. 

So, Will the test_data include the the true  Day and timestamp values, or they will be aligned to  Day 1 and timestamp 00:00 regardless of the true timestamp.  
<img width="661" alt="Screen Shot 2022-05-30 at 5 36 06 PM" src="https://user-images.githubusercontent.com/7350222/170968328-0d563c76-70d8-44cf-bd4b-c504da5f3023.png">

**Comment by zhoujingbo** [comment, marked as answer]
Hi,

The timestamp will be aligned to Day 1  regardless of the true dates.  The test data will start from day 1.

However, we keep the periodic information within a day. In other words, if the starting timestamp of the testing data is 8:00 AM, the true timestamp of the data point in real life is also 8:00 AM. 

Thanks a lot.
